app/policies/work_policy.rb

```diff
@@ -28,9 +28,8 @@ def collection_manager?
   end
 
   def collection_depositor?
-    return record.depositors.include?(user) if record.is_a? Collection
-
-    record.collection.depositors.include?(user)
```

Comment:
This was dead code. `WorksController#create` *only* authorizes collections, not works, because there is no work to authorize yet.